frontend/src/lib/components/TimeSensitiveAuthentication/TimeSensitiveAuthentication.tsx

```diff
@@ -20,14 +20,16 @@ export function TimeSensitiveAuthenticationModal(): JSX.Element {
             onClose={() => setDismissedReauthentication(true)}
             maxWidth="30rem"
             footer={
-                <LemonButton
-                    type="primary"
-                    form="reauthentication"
-                    loading={isReauthenticationSubmitting}
-                    onClick={submitReauthentication}
-                >
-                    Re-authenticate
-                </LemonButton>
+                user?.has_social_auth ? undefined : (
```

Comment:
Oooh actually nope. This should be based on whether they `has_password` instead as otherwise there will be a password input but no button to submit

Reply:
ah, you can have both? 🙈 👍